RiotSwiftUI/Modules/Common/Test/XCTestPublisherExtensions.swift

```diff
@@ -0,0 +1,55 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import XCTest
+import Combine
+
+/**
+ XCTest utility to wat for results from publishers, so that the output can be used for assertions.
+ */
+@available(iOS 14.0, *)
+extension XCTestCase {
+    func xcAwait<T: Publisher>(
```

Comment:
This is a nice addition! I think it would be worthwhile having a small example code block in a doc comment for the function.